policyengine_us/parameters/gov/irs/gross_income/sources.yaml

```diff
@@ -3,6 +3,9 @@ values:
   2010-01-01:
     - employment_income
     - self_employment_income
+    - partnership_s_corp_income
+    - farm_income
```

Comment:
`farm_income` is in the CPS data and its documentation says this:
> Income from agricultural businesses. Do not include this income in self-employment income.

A recursive grep from the top of the source code tree shows that it is an independent variable, not included in any other income type except `qualified_business_income` (as it should be).

`farm_rent_income` also seems to be independent.

These two variables have been included ever since the TaxCalculator US income tax code was ported over to PolicyEngineUS.  They are independent variables in the IRS PUF dataset that is used by TaxCalculator.